lib/hci-socket/hci.js

```diff
@@ -86,7 +87,7 @@ Hci.prototype.init = function() {
 
   var deviceId = process.env.NOBLE_HCI_DEVICE_ID ? parseInt(process.env.NOBLE_HCI_DEVICE_ID, 10) : undefined;
 
-  this._socket.bindRaw(deviceId);
+  this._deviceId = this._socket.bindRaw(deviceId);
```

Comment:
Hi @vdharmon,

From my understanding, `bindRaw` does not return anything, should this line be `this._deviceId = deviceId;`?